src/bootstrap/config.rs

```diff
@@ -174,7 +180,9 @@ struct Build {
 }
 
 /// TOML representation of various global install decisions.
-#[derive(RustcDecodable, Default, Clone)]
+#[derive(Deserialize, Default, Clone)]
+#[serde(deny_unknown_fields)]
+#[serde(rename_all = "kebab-case")]
```

Comment:
Could this be specified as `#[serde(rename_all = "kebab-case", deny_unknown_fields)]`?